developer/src/kmc-keyboard-info/src/min-keyman-version.ts

```diff
@@ -0,0 +1 @@
+export const minKeymanVersion = '5.0';
```

Comment:
```suggestion
//
// Minimum Keyman version that will be supplied for a .keyboard_info 
// if a compiled keyboard does not list a minimum version itself.
//
export const minKeymanVersion = '5.0';
```

Reply:
```suggestion
/**
 * Minimum Keyman version that will be supplied for a .keyboard_info 
 * if a compiled keyboard does not list a minimum version itself.
 */
export const minKeymanVersion = '5.0';
```

This way, it shows up via Intellisense on imported references.